cpp/include/cudf/aggregation.hpp

```diff
@@ -174,8 +174,13 @@ make_nunique_aggregation(include_nulls _include_nulls = include_nulls::NO);
  * @brief Factory to create a `nth_element` aggregation
  * 
  * `nth_element` returns the n'th element of the group/series.
+ *
+ * If @p n is not within the range [-group_size, group_size), the result of the
+ * respective group will be null. Negative indices starts from end of the group.
```

Comment:
This needs a more formal definition. What does "start from the end of the group" mean?

Reply:
definition for negative indices updated